auth/basics/android.md

```diff
@@ -230,9 +249,9 @@ skygear.saveUser(newUser, new UserSaveResponseHandler() {
 });
 ```
 
-If you only want to change one of the user field, just replace the new username/email with `currentUser.getUsername()`/`currentUser.getEmail()`
+If you only want to change one of the user fields, just replace the new username/email with `currentUser.getUsername()`/`currentUser.getEmail()`.
 
-### Change the password of a user
+## Updating the password of a user
```

Comment:
@Camerash I saw there's a section in JS about updating the password of a user https://docs.skygear.io/guides/auth/basics/js/#changing-the-username-and-email-of-a-user

Do we have this API in Android too?